homeassistant/components/flux_led/light.py

```diff
@@ -34,7 +36,9 @@
 
 DOMAIN = "flux_led"
 
-SUPPORT_FLUX_LED = SUPPORT_BRIGHTNESS | SUPPORT_EFFECT | SUPPORT_COLOR
+SUPPORT_FLUX_LED = (
+    SUPPORT_BRIGHTNESS | SUPPORT_EFFECT | SUPPORT_COLOR | SUPPORT_COLOR_TEMP
+)
```

Comment:
Did you mistakenly add `SUPPORT_COLOR_TEMP` to both here and in the `supported_features` list separately?